plugins/outputs/sql/README.md

```diff
@@ -187,11 +188,8 @@ docs](https://modernc.org/sqlite) for details.
 
 #### DSN
 
-Currently, Telegraf's sql output plugin depends on
-[clickhouse-go v1.5.4](https://github.com/ClickHouse/clickhouse-go/tree/v1.5.4)
-which uses a [different DSN
-format](https://github.com/ClickHouse/clickhouse-go/tree/v1.5.4#dsn) than its
-newer `v2.*` version.
+Note that even when the DSN is specified as `https://` the `secure=true`
+parameter is still required.  
 
```

Comment:
How about adding a warning here that the DSN now should adhere to the `v2` schema including a link to how a DSN might be constructed? Furthermore, add a brief note about the fact that 

> The plugin tries to convert v1 DSNs to v2 but as both schemata are not fully equivalent some parameters might not work anymore. Please check for warnings in your log file and refer to the [DSN v2 documentation](url) for available options

Reply:
Done.